clients/web/src/views/body/FolderView.js

```diff
@@ -19,7 +19,12 @@ var FolderView = View.extend({
         this.folderEdit = settings.folderEdit || false;
         this.itemCreate = settings.itemCreate || false;
 
+        this.render();
+    },
+
+    render: function () {
         this.hierarchyWidget = new HierarchyWidget({
```

Comment:
Usually we try to avoid creating the child widget on each render call of the parent widget. Why this change?